jmclient/jmclient/wallet_rpc.py

```diff
@@ -582,6 +599,57 @@ def preflight(self, request):
         request.setHeader("Access-Control-Allow-Methods", "POST")
 
     with app.subroute(api_version_string) as app:
+        @app.route('/token', methods=['POST'])
+        def refresh(self, request):
+            self.check_cookie(request, verify_exp=False)
+
+            def _mkerr(err, description=""):
+                return make_jmwalletd_response(
+                    request, status=400, message=err, error_description=description
```

Comment:
Shouldn't some (or all?) of the places where this is called, be using 401 and not 400?

Reply:
Authorization happens when verifying bearer token, if this verification fails a 401 will be returned.
If refresh token verification fails, it is a request error, 400, in line with https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-08#name-error-response

Reply:
Read and understood, thanks.